core/sr-primitives/src/generic/checked_extrinsic.rs

```diff
@@ -55,3 +56,15 @@ where
 		(self.function, self.signed.map(|x| x.0))
 	}
 }
+
+impl<AccountId, Index, Call> WeighableCall
+	for CheckedExtrinsic<AccountId, Index, Call>
```

Comment:
would exceed 80 chars.

Reply:
80 is good, but just in case here is an excerpt from our style-guide:

> Lines should be longer than 100 characters long only in exceptional circumstances and certainly no longer than 120. For this purpose, tabs are considered 4 characters wide.